Source/diffusion/Castro_diffusion.cpp

```diff
@@ -173,6 +173,98 @@ Castro::getTempDiffusionTerm (Real time, MultiFab& state_in, MultiFab& TempDiffT
        FillPatch(getLevel(level-1),CrseTemp,1,time,State_Type,UTEMP,1);
    }
 
-   diffusion->applyop(level, Temperature, CrseTemp, TempDiffTerm, coeffs);
+   if (diffuse_use_amrex_mlmg) {
+       // Evaluates ∇ ⋅(k_th ∇T) using AMReX
+       diffusion->applyop(level, Temperature, CrseTemp, TempDiffTerm, coeffs);
+   } else {
+       // Evaluates ∇ ⋅(k_th ∇T) without using AMReX
+#ifdef AMREX_USE_OMP
+#pragma omp parallel if (Gpu::notInLaunchRegion())
+#endif
+       for (MFIter mfi(Temperature, TilingIfNotGPU()); mfi.isValid(); ++mfi)
+       {
+           const auto dx = geom.CellSizeArray();
+           const auto dxinv = geom.InvCellSizeArray();
+           const auto problo = geom.ProbLoArray();
+           const int coord = geom.Coord();
+
+           const Box& bx = mfi.tilebox();
+           Array4<Real const> const& Temp_array = Temperature.array(mfi);
+           Array4<Real> const & TempDiff_array = TempDiffTerm.array(mfi);
 
+           ParallelFor(bx,
+           [=] AMREX_GPU_DEVICE (int i, int j, int k)
+           {
+               // edged based k_th in different averaging direction.
+               Vector<Array4<Real const> const> edge_coeff_arrs {AMREX_D_DECL((*coeffs[0]).array(mfi),
```

Comment:
the cpplinter doesn't like this

Reply:
I think this needs to be outside of the ParallelFor

Reply:
yup I fixed it